### ISSUE TITLE
Can't understand the output of layer in Xception.py

### ISSUE THREAD

**Issue by alyato** [fchollet/keras#6072]
hi,guys. I check the code `Xception.py`in [https://github.com/fchollet/keras/blob/master/keras/applications/xception.py](url) ,but can't understand the output of `residual`and ` x = layers.add([x, residual])`

>...
    residual = Conv2D(128, (1, 1), strides=(2, 2),padding='same', use_bias=False)(x)
    residual = BatchNormalization()(residual)
    x = SeparableConv2D(128, (3, 3), padding='same', use_bias=False, name='block2_sepconv1')(x)
    x = BatchNormalization(name='block2_sepconv1_bn')(x)
    x = Activation('relu', name='block2_sepconv2_act')(x)
    x = SeparableConv2D(128, (3, 3), padding='same', use_bias=False, name='block2_sepconv2')(x)
    x = BatchNormalization(name='block2_sepconv2_bn')(x)
    x = MaxPooling2D((3, 3), strides=(2, 2), padding='same', name='block2_pool')(x)
    x = layers.add([x, residual])

i don't know  its mean when setting `strides=(2,2),padding=same` in `Conv2D` and `Maxpool2D`
So i think that 
    

> x = MaxPooling2D((3, 3), strides=(2, 2), padding='same', name='block2_pool')(x)
the output is (128,73,73)
after the `x = layers.add([x, residual])`,i don't understand the output of `x`

Can some guys give me some advices? Thanks.
